lib/praxis/action_definition.rb

```diff
@@ -31,7 +31,7 @@ def self.decorate_docs(&callback)
     end
 
     def initialize(name, resource_definition, **opts, &block)
-      @name = name
+      @name = name.to_sym
```

Comment:
I forget, but it was...actually, I think it might have been in a spec now that I think about it